packages/test-cases/cases/bind/sanity/imports-local/common.graphql

```diff
@@ -14,8 +14,6 @@ type CustomType {
   optBigint: BigInt
   bignumber: BigNumber!
   optBignumber: BigNumber
-  json: JSON!
```

Comment:
We should be keeping JSON as a base type

Reply:
This will conflict with the new JSON serializer. We will add it back with the `Json` name in a separate PR which will be the alias for the string.

Reply:
alternative would be to keep the old serializer for now and add new serializer as alias 

`import {JSON as Json} from "@serial-as/json"` and rename it in the next PR.

Reply:
Yes I think this is the best approach, since this change should only affect AssemblyScript specific code.

Reply:
I'm going to go through and revert these changes now.